tests/integration/orcid/test_orcid_utils.py

```diff
@@ -195,3 +199,10 @@ def test_orcids_for_push_orcid_in_author_with_claim(author_in_isolated_app):
 
     assert validate(record, 'hep') is None
     assert list(get_orcids_for_push(record)) == ['0000-0002-1825-0097']
+
+
+def test_get_literature_recids_for_orcid(isolated_app):
```

Comment:
You are only testing the happy flow. What about cases like when there is no PID or there are multiple PIDs or there is no 'ids' field at all, or the 'ids' field contains no ORCID schema or the 'ids' field contains the ORCID schema but the value is empty, etc

Reply:
Whoops, absolutely! Will add all these tests.

Reply:
For the record, I added below tests for:
- there is no PID
- there are multiple PIDs
- there is no 'ids' field at all
- the 'ids' field contains no ORCID schema

but we agreed that don't need to write a test for 
- the 'ids' field contains the ORCID schema but the value is empty

as this is disallowed by the schema.